crates/messages/proto/messages.proto

```diff
@@ -31,137 +19,149 @@ message WorkerState {
 }
 
 message DatasetRanges {
+  option deprecated = true;
```

Comment:
Not yet, unfortunately, because we have to keep the scheduler at the state when it still communicates with both the old and the new workers. If you want, I can submit a PR with the final state of the protobufs after the transition is made.